03/app.js

```diff
@@ -1,14 +1,34 @@
-console.log('DOM');
+console.log("DOM");
 
 const buttonSettings = {
-    attr: {
-        className: 'btn',
-        title: 'super button'
-    },
-    css: {
-        border: '1px solid #336699',
-        padding: '5px 20px',
-        color: '#444'
-    },
-    text: 'Click me!',
-}
\ No newline at end of file
+  attr: {
+    className: "btn",
+    title: "super button",
+  },
+  css: {
+    border: "1px solid #336699",
+    padding: "5px 20px",
+    color: "#444",
+  },
+  text: "Click me!",
+};
+const section = document.querySelector(".parent-for-button");
+const button = document.createElement("button");
+section.appendChild(button);
+
+for (const key in buttonSettings) {
```

Comment:
W zasadzie i tak rozdzieliłeś całość if-ami na "kilka" `for...in` więc lepiej byłoby ich nie zagnieżdzać:
```
for(const property in buttonSettings.css) { ...
```

oraz 
```
for(const property in buttonSettings.attr) { ...
```